src/scaffold/template/base/.eslintrc.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
This is a suggestion for a default ruleset. This can be overwritten using a apikana-defaults template, right?

Reply:
yes